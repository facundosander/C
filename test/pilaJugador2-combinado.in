crearPilaJugadores

imprimirCantidadPila

crearJugada
1 1 2 3 4
crearPartida
agregarEnPartida
crearJugador 1 21 Uruguay Juan

apilarPilaJugadores
liberarJugador

imprimirCantidadPila

crearJugada
2 2 2 2 2
crearPartida
agregarEnPartida
crearJugador 7 20 Uruguay Nicol√°s

apilarPilaJugadores
liberarJugador

imprimirCantidadPila

crearJugada
3 4 3 2 1
crearPartida
agregarEnPartida
crearJugador 4 23 Uruguay Jessica

apilarPilaJugadores
liberarJugador

imprimirCantidadPila

obtenerCimaPila

desapilar

imprimirCantidadPila

obtenerCimaPila

desapilar

imprimirCantidadPila

obtenerCimaPila

desapilar

liberarPilaJugadores

Fin
