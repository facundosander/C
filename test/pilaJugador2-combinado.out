1>2>La cantidad de jugadores en la pila es: 0.
3>4>5>6>7>8>9>La cantidad de jugadores en la pila es: 1.
10>11>12>13>14>15>16>La cantidad de jugadores en la pila es: 2.
17>18>19>20>21>22>23>La cantidad de jugadores en la pila es: 3.
24>Jugador 4:  Jessica, Uruguay, 23 a침os
Jugada: 3 
Movimientos: Abajo Arriba Izquierda Derecha 
25>26>La cantidad de jugadores en la pila es: 2.
27>Jugador 7:  Nicol치s, Uruguay, 20 a침os
Jugada: 2 
Movimientos: Izquierda Izquierda Izquierda Izquierda 
28>29>La cantidad de jugadores en la pila es: 1.
30>Jugador 1:  Juan, Uruguay, 21 a침os
Jugada: 1 
Movimientos: Derecha Izquierda Arriba Abajo 
31>32>33>Fin.
